lib/view/exhibition_detail_preview.dart

```diff
@@ -45,50 +47,27 @@ class ExhibitionPreview extends StatelessWidget {
             title: exhibition.title,
             padding: const EdgeInsets.symmetric(vertical: 20),
           ),
-          if (exhibition.curator != null) ...[
-            Text('curator'.tr(), style: subTextStyle),
+          if (listCurators.isNotEmpty) ...[
+            Text(
+              exhibition.curators != null ? 'curators'.tr() : 'curator'.tr(),
```

Comment:
As we are displaying base on listCurators, i think we should check `listCurators.length > 1`